libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -1245,6 +1264,36 @@ impl NodeAPI for Greenlight {
         }
     }
 
+    async fn start_keep_alive(&self, mut shutdown: watch::Receiver<()>) {
+        info!("keep alive started");
+        let mut client = self.get_node_client().await.unwrap();
```

Comment:
Is there a safer way to handle this unwrap to prevent a panic?

Reply:
Of course, good catch. Was just a left over from my test. will fix now.

Reply:
Done